ios/RNBLEPrinter.m

```diff
@@ -20,90 +27,49 @@ - (dispatch_queue_t)methodQueue
 
 RCT_EXPORT_MODULE()
 
-RCT_EXPORT_METHOD(init:(RCTResponseSenderBlock)successCallback
+RCT_EXPORT_METHOD(connectAndSend:(NSString *)bdAddress
+                  printRawData:(NSString *)text
+                  success:(RCTResponseSenderBlock)successCallback
                   fail:(RCTResponseSenderBlock)errorCallback) {
     @try {
-        _printerArray = [NSMutableArray new];
-        m_printer = [[NSObject alloc] init];
-        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleNetPrinterConnectedNotification:) name:@"NetPrinterConnected" object:nil];
-        // API MISUSE: <CBCentralManager> can only accept this command while in the powered on state
-        [[PrinterSDK defaultPrinterSDK] scanPrintersWithCompletion:^(Printer* printer){}];
-        successCallback(@[@"Init successful"]);
-    } @catch (NSException *exception) {
-        errorCallback(@[@"No bluetooth adapter available"]);
-    }
-}
+        int result = EPOS2_SUCCESS;
 
-- (void)handleNetPrinterConnectedNotification:(NSNotification*)notification
-{
-    m_printer = nil;
-}
-
-RCT_EXPORT_METHOD(getDeviceList:(RCTResponseSenderBlock)successCallback
-                  fail:(RCTResponseSenderBlock)errorCallback) {
-    @try {
-        !_printerArray ? [NSException raise:@"Null pointer exception" format:@"Must call init function first"] : nil;
-        [[PrinterSDK defaultPrinterSDK] scanPrintersWithCompletion:^(Printer* printer){
-            [_printerArray addObject:printer];
-            NSMutableArray *mapped = [NSMutableArray arrayWithCapacity:[_printerArray count]];
-            [_printerArray enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
-                NSDictionary *dict = @{ @"device_name" : printer.name, @"inner_mac_address" : printer.UUIDString};
-                [mapped addObject:dict];
-            }];
-            NSMutableArray *uniquearray = (NSMutableArray *)[[NSSet setWithArray:mapped] allObjects];;
-            successCallback(@[uniquearray]);
-        }];
-    } @catch (NSException *exception) {
-        errorCallback(@[exception.reason]);
-    }
-}
+        printer_ = [[Epos2Printer alloc] initWithPrinterSeries:1 lang:EPOS2_MODEL_ANK];
```

Comment:
PrinterSeries is set to 1, what does it mean?

Reply:
It has no functionality effects. It is the exact printer model.